content/6-network-security/4-fully-private-cluster/2-install-alb-controller-addon/index.en.md

```diff
@@ -0,0 +1,187 @@
+---
+title : "Installing the AWS Load Balancer Controller add-on"
+weight : 152
+---
+
+
+The AWS Load Balancer Controller manages AWS Elastic Load Balancers for a Kubernetes cluster. The controller provisions the following resources:
+
+Kubernetes Ingress
+
+    The AWS Load Balancer Controller creates an AWS Application Load Balancer (ALB) when you create a Kubernetes Ingress.
+Kubernetes service of the LoadBalancer type
+
+    The AWS Load Balancer Controller creates an AWS Network Load Balancer (NLB) when you create a Kubernetes service of type LoadBalancer. In the past, the Kubernetes network load balancer was used for instance targets, but the AWS Load balancer Controller was used for IP targets. With the AWS Load Balancer Controller version 2.3.0 or later, you can create NLBs using either target type. For more information about NLB target types, see Target type in the User Guide for Network Load Balancers.
+
+The AWS Load Balancer Controller was formerly named the AWS ALB Ingress Controller. It's an open-source project
+managed on GitHub.
+
+# Create an IAM policy
+
+Create an IAM OIDC identity provider for your cluster with the following command
+
+
+```bash
+eksctl utils associate-iam-oidc-provider --region=$AWS_REGION --cluster=eksworkshop-eksctl-private --approve
+```
+
+
+Download an IAM policy for the AWS Load Balancer Controller that allows it to make calls to AWS APIs on your behalf from the Cloud9 instance.
+
+
+```bash
+curl -O https://raw.githubusercontent.com/kubernetes-sigs/aws-load-balancer-controller/v2.5.4/docs/install/iam_policy.json
```

Comment:
maybe export LBC version in a var, you use different version here : 2.5.4 and later 2.6.2. the latest is also 2.8.1

Reply:
Made the versions consistent